_posts/en/2021-06-08-working-in-the-open-part-2.md

```diff
@@ -0,0 +1,155 @@
+---
+layout: post
+title:  "Working in the Open: Part 2"
+ref: working-in-the-open-p2
+lang: en
+author: "Guillaume Charest, in collaboration with the IT Strategy team"
+date: "2021-06-08"
+excerpt_separator: <!--more-->
+---
+This series of blog posts will explore the concept of "Working in the Open".
+In this second entry, we will attempt to explain how you can get started while respecting your duties as a public servant!
```

Comment:
Suggest adding somewhere in the blog that public servant HAVE policy cover: The Directive on Open Government
https://www.tbs-sct.gc.ca/pol/doc-eng.aspx?id=28108

But that unfortunately it is too often seen as a CIO thing instead of a value and ethics thing.

Reply:
Good point!

Reply:
EA also should ["publish in the open all reusable common business capabilities, processes and enterprise solutions for others to develop and leverage cohesive horizontal enterprise services"](https://wiki.gccollab.ca/GC_Enterprise_Architecture/Framework)

Reply:
Thanks for the note @tsanov ! Will add it 😄

Reply:
> 
> 
> EA also should ["publish in the open all reusable common business capabilities, processes and enterprise solutions for others to develop and leverage cohesive horizontal enterprise services"](https://wiki.gccollab.ca/GC_Enterprise_Architecture/Framework)

@tsanov proving working in the open works!!!
Thanks for following our repo and contributing to it!!